runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetOverlay.java

```diff
@@ -57,11 +56,11 @@
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (config.netOverlay())
+		if (config.netOverlay() && plugin.isInDriftNetArea())
 		{
 			renderNet(graphics);
 		}
-		if (config.highlightFish())
+		if (config.highlightFish() && plugin.isInDriftNetArea())
```

Comment:
The `isInDriftNetArea` check should be its own thing, optionally as a Guard clause
```java
if (plugin.isInDriftNetArea())
{
	if (config.netOverlay())
	{
		renderNet(graphics);
	}
	if (config.highlightFish())
	{
		renderFishes(graphics);
	}
}
```
or
```java
if (!plugin.isInDriftNetArea())
{
	return null;
}
if (config.netOverlay())
{
	renderNet(graphics);
}
if (config.highlightFish())
{
	renderFishes(graphics);
}
```